src/pages/organize/[orgId]/campaigns/[campId]/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { GetServerSideProps } from 'next';
+
+
+import getCampaign from '../../../../../fetching/getCampaign';
+import getCampaignEvents from '../../../../../fetching/getCampaignEvents';
+import getOrg from '../../../../../fetching/getOrg';
+import OrganizeCampaignLayout from '../../../../../components/layout/OrganizeCampaignLayout';
+import { PageWithLayout } from '../../../../../types';
+import { scaffold } from '../../../../../utils/next';
```

Comment:
These relative imports are starting to look silly! I've added a task to #147 so that we can look into getting around this at some point. 😊